### PR TITLE
Storage product page

### PR DESCRIPTION
**Pull request by elad2412**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/lib/components/MultiCodeContextless.svelte [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    }
+    let copyText = CopyStatus.Copy;
+    async function handleCopy() {
+        await copy(content);
+
+        copyText = CopyStatus.Copied;
+        setTimeout(() => {
+            copyText = CopyStatus.Copy;
+        }, 1000);
+    }
```
Comment (ernstmul) [inline src/lib/components/MultiCodeContextless.svelte:37]: Same comment as in https://github.com/appwrite/website/pull/1485#discussion_r1848548939

File: src/routes/products/storage/(components)/OpenSource.svelte [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            content: 'Lorem ipsum dolor sit amet consectetur. Nullam ligula blandit.',
+            icon: '/images/icons/gradients/lock.svg'
+        }
+    ];
+</script>
+
+<div class="web-big-padding-section-level-2 light">
+    <div class="container">
+        <div class="web-hero is-center web-u-max-width-580 mx-auto">
+            <h2 class="web-display web-u-color-text-primary">Powered by open source</h2>
```
Comment (ernstmul) [inline src/routes/products/storage/(components)/OpenSource.svelte:26]: I see we type here "open source", on the site I also see "open-source" and "Open Source", do we want to use the same way everywhere? cc @LauraDuRy

File: src/routes/products/storage/+page.svelte [hunk 3]
```diff
@@ -0,0 +1,372 @@
...
+
+<style lang="scss">
+    .l-drop-shadow {
+        filter: drop-shadow(0 16px 32px rgb(55 59 77 / 0.04));
+    }
+
+    .hero {
+        display: grid;
+        grid-template-columns: 30.0625rem minmax(0, 1fr);
+        gap: 5vw;
```
Comment (ernstmul) [inline src/routes/products/storage/+page.svelte:261]: Does it make sense to do the gap in vw? Whats the use case?

File: src/routes/products/storage/+page.svelte [hunk 4]
```diff
@@ -0,0 +1,372 @@
...
+
+            > :first-child {
+                max-width: 40rem;
+                margin-inline: auto;
+            }
+
+            > :nth-child(2) {
+                scale: 1;
+                width: 100%;
+                //height: 24rem;
```
Comment (ernstmul) [inline src/routes/products/storage/+page.svelte:300]: comment can be removed

File: src/routes/products/storage/+page.svelte [hunk 5]
```diff
@@ -0,0 +1,372 @@
...
+    }
+
+    .e-bg-image-pattern-bottom {
+        background: url('/images/pages/storage/pattern-1.png') no-repeat center bottom;
+        background-size: 1500px;
+        @media (max-width: 650px) {
+            background-position: center bottom -50%;
+        }
+        @media (max-width: 1198px) {
+            //background-position: center bottom -100px;
```
Comment (ernstmul) [inline src/routes/products/storage/+page.svelte:313]: comment can be removed

File: src/routes/products/storage/EventCard.svelte [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+        <p class="web-sub-body-500">
+            {description}
+        </p>
+        <div class="u-flex u-flex-wrap u-gap-8 u-padding-block-start-16 mbs-auto">
+            <button class="web-button is-secondary">
+                <span>{buttonText}</span>
+            </button>
+            <!-- <button class="web-button is-text">
+        <span>Add to calendar</span>
+      </button> -->
```
Comment (ernstmul) [inline src/routes/products/storage/EventCard.svelte:61]: comment can be removed

File: src/lib/components/MultiCodeContextless.svelte (hunk 1 above)
Comment (thejessewinton) [reply src/lib/components/MultiCodeContextless.svelte:37]: 👍🏼

File: src/routes/products/storage/(components)/OpenSource.svelte (hunk 2 above)
Comment (thejessewinton) [reply src/routes/products/storage/(components)/OpenSource.svelte:26]: Good question!

Comment (thejessewinton) [reply src/routes/products/storage/(components)/OpenSource.svelte:26]: Going to get this shipped for now, can edit later.

**Review by loks0n — APPROVED** [review]

**Pull request merged**